sources/background_scripts/notifications.js

```diff
@@ -0,0 +1,25 @@
+NewTabTweaker = {};
+NewTabTweaker.Notifications = {
+	notification_ids: [],
```

Comment:
This should be a variable, not a property, and should be renamed to `active_ids` to better illustrate its purpose.